src/kbc_modules/cc_kbc/attester/cca/mod.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2023 Arm Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::Attester;
+use anyhow::*;
+use std::env;
+
+
+// If the environment variable "CCA_ATTESTER" is set,
+// the TEE platform is considered as "CCA".
+pub fn detect_platform() -> bool {
+    env::var("CCA_ATTESTER").is_ok()
```

Comment:
Who sets this? Is there a way to query the platform more directly like via cpuid?

Reply:
I was thinking this might be an env, just like `AA_SAMPLE_ATTESTER_TEST`, query from platform is a great idea. Let's see if we can move toward that.